public/electron.js

```diff
@@ -129,6 +129,24 @@ function createWindow() {
   });
 }
 
+ipcMain.on('restart-lnd-process', async event => {
+  lndProcess && lndProcess.kill('SIGINT');
+  let restartError;
+  try {
+    lndProcess = await startLndProcess({
+      isDev,
+      lndSettingsDir,
+      lndPort: LND_PORT,
+      lndPeerPort: LND_PEER_PORT,
+      logger: Logger,
+      lndArgs,
+    });
+  } catch (err) {
+    restartError = err;
+  }
+  event.sender.send('lnd-restart-error', { restartError });
+});
+
```

Comment:
Not sure I understand why we need to restart lnd. Wouldn't it just be easier to restart the app?